build-a-bot/src/store/index.ts

```diff
@@ -0,0 +1,14 @@
+import { createStore } from 'vuex';
+import Cart from '@/data/Cart';
+import { state, State } from './state';
+
+const store = createStore<State>({
+  state,
+  mutations: {
+    addRobotToCart(s: State, robot: Cart) {
```

Comment:
@Jaxwood  remove `s: State,` and compile -> you get an error messae. Please explain why so and/or how to fix it?